src/bufrlib.h.in

```diff
@@ -49,6 +47,63 @@ int icvidx(int ii, int jj, int numjj);
  * given time for a Table B descriptor.*/
 #define MXNAF @MXNAF@
 
+/**
+ * Convert an FXY value from its bit-wise (integer) representation to its
```

Comment:
WE are calling this a "WMO bit-wise representation of the FXY value" and adding a reference to the ifxy() function, which explains it fully. See adn30.f for example.

Reply:
I know we discussed changing this description, but from what I recall we were going to think some more about what a suitable replacement would be, and I didn't think we'd decided on anything yet.

That said, I'm OK with this new description, so I'll go ahead and fix that.